build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -520,6 +520,49 @@ message Platform {
   // The server MAY use the `value` of one or more properties to determine how
   // it sets up the execution environment, such as by making specific system
   // files available to the worker.
+  //
+  // The following standard `name`s are defined:
```

Comment:
I'm wondering if this really belongs in the core API file, or in a separate lexicon file. That would allow the API to remain generic and flexible, but clients and servers could choose to specify that they are using a particular platform lexicon (which could then have well-defined semantics for how particular fields are matched).

Reply:
Would that separate lexicon be in a separate repository or remain here and thus, still be considered (an optional) part of the official API specification?

I don't have any objections moving this to a separate file. However, I think we should only have one upstream platform lexicon. It may make sense for special purpose or internal deployments of remote API infrastructure to have a different platform lexicon. However, I don't see a point in defining multiple incompatible lexicons for general purpose build infrastructure.

Reply:
I've moved this to a separate file called `platform.md`. I'm open for better suggestions on name or format.